src/vs/workbench/browser/parts/titlebar/media/titlebarpart.css

```diff
@@ -230,8 +230,7 @@
 	margin-left: 4px;
 }
 
-.monaco-workbench .part.titlebar > .titlebar-container.counter-zoom .menubar .menubar-menu-button > .menubar-menu-items-holder.monaco-menu-container,
-.monaco-workbench .part.titlebar > .titlebar-container.counter-zoom .monaco-toolbar .dropdown-action-container {
+.monaco-workbench .part.titlebar > .titlebar-container.counter-zoom .menubar .menubar-menu-button > .menubar-menu-items-holder.monaco-menu-container {
```

Comment:
What is the reason for removing this? This was added to fix the zooming for the dropdown action to the right of the command center. Without this, the zooming is wrong for that action. This line counter zooms the position because the title bar does not support negative zoom values (due to WCO).

Ref: https://github.com/microsoft/vscode/pull/231825